data-serving/data-service/schemas/cases.schema.json

```diff
@@ -73,9 +70,21 @@
           "administrativeAreaLevel2": {
             "bsonType": "string"
           },
-          "locality": {
+          "administrativeAreaLevel3": {
+            "bsonType": "string"
+          },
+          "place": {
```

Comment:
Making sure I grok -- e.g., "Boston Children's Hospital"?

Reply:
I think this is supposed to be a human readable name of the most granular feature?

Reply:
Yes, per the spec, "Specific place location" -- so I think this would be a hospital, etc.

Reply:
Per our convo offline:

"place": specific location, e.g. hospital; cannot currently get from mapbox API, but it would map to POI; curators will be able to input it in a text field, and imported and/or parsed cases may have it
"name": human-readable name of the location, ironically to be populated by mapbox's place field